src/preamble.js

```diff
@@ -1228,32 +1228,23 @@ if (typeof SharedArrayBuffer !== 'undefined') {
         var num = end - begin;
         var target = new ArrayBuffer(num);
         var targetArray = new Uint8Array(target);
-        var sourceArray = new SharedUint8Array(this, begin, num);
+        var sourceArray = new Uint8Array(this, begin, num);
         targetArray.set(sourceArray);
         return target;
       };
     }
   })();
-
-  HEAP8 = new SharedInt8Array(buffer);
-  HEAP16 = new SharedInt16Array(buffer);
-  HEAP32 = new SharedInt32Array(buffer);
-  HEAPU8 = new SharedUint8Array(buffer);
-  HEAPU16 = new SharedUint16Array(buffer);
-  HEAPU32 = new SharedUint32Array(buffer);
-  HEAPF32 = new SharedFloat32Array(buffer);
-  HEAPF64 = new SharedFloat64Array(buffer);
 } else {
   if (!ENVIRONMENT_IS_PTHREAD) buffer = new ArrayBuffer(TOTAL_MEMORY);
-  HEAP8 = new Int8Array(buffer);
-  HEAP16 = new Int16Array(buffer);
-  HEAP32 = new Int32Array(buffer);
-  HEAPU8 = new Uint8Array(buffer);
-  HEAPU16 = new Uint16Array(buffer);
-  HEAPU32 = new Uint32Array(buffer);
-  HEAPF32 = new Float32Array(buffer);
-  HEAPF64 = new Float64Array(buffer);
 }
+HEAP8 = new Int8Array(buffer);
```

Comment:
this can now be replaced with a call to `updateGlobalBufferViews` (recently added in this file)